view/frontend/web/js/model/fastlane.js

```diff
@@ -192,32 +166,112 @@ define([
             }
             require.config({
                 paths: {
-                    bold_paypal_fastlane: 'https://www.paypal.com/sdk/js?client-id=' + gatewayData.client_id + '&components=fastlane' + debugMode
+                    bold_paypal_fastlane: 'https://www.paypal.com/sdk/js?client-id=' + gatewayData.client_id + '&components=buttons,fastlane' + debugMode,
                 },
-                shim: {
-                    'bold_paypal_fastlane': {
-                        exports: 'paypal.fastlane'
-                    }
+            });
+            await new Promise((resolve, reject) => {
+                require(['bold_paypal_fastlane'], resolve, reject);
+            });
+
+            window.boldFastlaneInstance = await window.paypal.Fastlane();
+        },
+        /**
+         * Load Axo script with require js.
+         *
+         * @param {{client_token: string}} gatewayData
+         * @return {void}
+         */
+        loadAxo: function (gatewayData) {
```

Comment:
This function doesn't really `loadAxo` anymore, I would suggest renaming to help illustrate what this method really does.

Reply:
Renamed.